doc/lua_api.txt

```diff
@@ -6409,18 +6409,36 @@ object you are working with still exists.
     * 7 - dig
     * 8 - place
     * 9 - zoom
-* `set_physics_override(override_table)`
-    * `override_table` is a table with the following fields:
+* `get_effective_physics()`: Returns a table representing the current player movement physics parameters.
+    * If physics modifiers are enabled, this will be the resolved values after the modifiers are applied.
+    * Otherwise, this will return same values as `get_physics_override()`, but without the flags.
+    * fields:
         * `speed`: multiplier to default walking speed value (default: `1`)
         * `jump`: multiplier to default jump value (default: `1`)
         * `gravity`: multiplier to default gravity value (default: `1`)
-        * `sneak`: whether player can sneak (default: `true`)
-        * `sneak_glitch`: whether player can use the new move code replications
-          of the old sneak side-effects: sneak ladders and 2 node sneak jump
-          (default: `false`)
-        * `new_move`: use new move/sneak code. When `false` the exact old code
-          is used for the specific old sneak behaviour (default: `true`)
-* `get_physics_override()`: returns the table given to `set_physics_override`
+* `set_physics_modifier(key_string, modifier_table)`: Adds a physics modifier to
+   the player under the given key.
+   * Modifiers are resolved by first adding all `add` modifiers, and then multiplying all `multiply` modifiers.
+   * `key_string`: A string key identifying this modifier. Should take the format
+     "modname:modifier_name".
+   * `modifier_table`: a table with the same fields as `get_effective_physics`. Plus:
+       * `type` - `multiply` or `add`. Default: `multiply`.
```

Comment:
Document that `add` is applied first?  (There's already confusion over this; Wuzzy above wrote how he thought that add was applied in the order it was set, which isn't how it goes.)